src/DataDictionary/table/DataDictionaryTable/index.js

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+import localconf from '../../../localconf';
```

Comment:
It might be better to just import `appname` since thats all thats being used from here